shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -867,7 +867,7 @@ - (UIAccessibilityTraits)accessibilityTraits {
 @end
 
 @interface FlutterPlatformViewSemanticsContainer ()
-@property(nonatomic, retain) UIView* platformView;
+@property(nonatomic, assign) UIView* platformView;
```

Comment:
See https://github.com/flutter/engine/pull/52484, platformView needs to be weak here.  I'll figure out a test for that one first and let that land.